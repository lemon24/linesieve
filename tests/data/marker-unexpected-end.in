--section '(\S+):$' --success ok --failure fail show $nothing
---
one:
1
two:
2
