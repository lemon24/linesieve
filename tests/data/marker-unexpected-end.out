..
two
2
unexpected end during two
